bootstrap/ansible/playbook.yml

```diff
@@ -56,17 +55,14 @@
               - libffi-devel
             state: present
 
-        - name: Install SCL
```

Comment:
Each time the SCL role is included, it runs a dozen or so tasks, so having it do everything in one run would be ideal to not have a bunch of redundant tasks. I could use the `tasks_from` option that bypasses `main.yml` to directly run specific playbooks, but I'm not sure if that's good practice. If I use `tasks_from`, it would skip over the install setuptools which is included in `main.yml`, and I'd have to include it in the `Install required packages` task. If that's fine, I can split the include into two, otherwise I can rename the task to mention postgres too
![image](https://user-images.githubusercontent.com/43729496/223881580-d9d0b243-17f0-4ff4-b025-612780f90219.png)

Reply:
Okay--can you just mention Postgres in the task and include a brief variation of what you explained:

> Each time the SCL role is included, it runs a dozen or so tasks, so having it do everything in one run would be ideal to not have a bunch of redundant tasks.